modules/popup-stack/lib/PopupStack.ts

```diff
@@ -246,7 +246,10 @@ export const PopupStack = {
    */
   remove(element: HTMLElement): void {
     // Find the stack the popup belongs to.
-    const stack = find(stacks, stack => !!find(stack.items, item => item.element === element));
+    const stack = find(
+      stacks,
+      stack => !!find(PopupStack.getElements(stack), el => el === element)
+    );
```

Comment:
It was a mistake to use `stack.items` since that only exists on non-adapted stacks. `PopupStack.getElements` should always return elements.